lib/facter/debugging_check.rb

```diff
@@ -0,0 +1,9 @@
+Facter.add('nvdebugging') do
+  setcode do
+    if File.exist? '/var/spool/slurmd/nvperfenabled'
```

Comment:
I don't think it is possible, but would be nice if you could reference `rofile_gpu::dcgm::telegraf::nv_debug_check` here instead of the hard coded file.

Reply:
Hiera doesn't seem to be able to resolve into facts. I tried this previously, but sadly, I couldn't figure it out.